core/metacling/src/TCling.cxx

```diff
@@ -1269,8 +1269,8 @@ TCling::TCling(const char *name, const char *title, const char* const argv[])
 #else
       const char EnvPathDelimiter = ':';
 #endif // R__WIN32
-      auto GetEnvVarPath = [EnvPathDelimiter](const std::string &EnvVar,
-                                              std::vector<std::string> &Paths) {
+      auto GetEnvVarPath = [](const std::string &EnvVar,
```

Comment:
actually the var should be renamed kEnvPathDelimiter since it is constant and probably made into a constexpr.